addons/calendar/data/mail_template_data.xml

```diff
@@ -262,97 +261,102 @@
-                </t>
-            </ul>
-        </td>
-    </tr></table>
-    <br/>
-    Thank you,
-    <t t-if="object.event_id.user_id.signature">
-        <br />
-        <div>--<br/><t t-out="object.event_id.user_id.signature or ''">Mitchell Admin</t></div>
-    </t>
+                    </t>
+                </td>
+            </tr>
+            <tr t-if="recurrent" style="padding-top: 6px;">
+                <td style="width: 100px; color: #676F7B;">When</td>
+                <td>
+                    <t t-out="object.recurrence_id.get_recurrence_name() or ''">Every 1 Weeks, for 3 events</t>
+                </td>
+            </tr>
+            <tr t-if="not object.event_id.allday and object.event_id.duration" style="padding-top: 6px;">
+                <td style="width: 100px; color: #676F7B;">Duration</td>
+                <td>
+                    <t t-out="('%dH%02d' % (object.event_id.duration,round(object.event_id.duration*60)%60)) or ''">0H30</t>
+                </td>
+            </tr>
+            <tr t-if="object.event_id.location" style="padding-top: 6px;">
+                <td style="width: 100px; color: #676F7B; vertical-align: top;">Location</td>
+                <td style="max-width: 450px;">
+                    <t t-out="object.event_id.location or ''">Bruxelles</t>
+                    <a target="_blank" t-if="not is_online or is_online and object.event_id.location != object.event_id.appointment_type_id.location_id.name" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}" style="margin-left: 12px; font-size: 12px; color: #008f8c; text-decoration: underline;">View Map</a>
+                </td>
+            </tr>
+            <tr t-if="object.event_id.videocall_location" style="padding-top: 6px;">
+                <td style="width: 100px; color: #676F7B;">
+                    <t t-if="object.get_base_url() in object.event_id.videocall_location">Join with</t>
+                    <t t-else="">Join</t>
+                </td>
+                <td>
+                    <a t-attf-href="object.event_id.videocall_location" target="_blank" style="color: #008f8c;">
+                        <t t-if="object.get_base_url() in object.event_id.videocall_location">Odoo Discuss</t>
+                        <t t-else="">Video meeting</t>
+                    </a>
+                </td>
+            </tr>
+        </table>
+    </div>
+    <div style="margin: 32px 0 0;">
+        <h2 t-attf-style="margin-bottom: 12px; font-size: 14px; font-weight: bold; color: {{user.company_id.email_secondary_color or '#875A7B'}};">Attendees</h2>
+        <div t-foreach="object.event_id.attendee_ids" t-as="attendee" style="margin-bottom: 5px;">
+            <img t-if="attendee.state" t-attf-src="/calendar/static/src/img/state_{{ attendee.state }}.png" style="border:0; outline:none; width: 15px; height: 15px; vertical-align: text-bottom;" />
+            <t t-if="attendee.common_name != object.common_name">
```

Comment:
Globally same remarks as for appointment PR, "outline" style key should be doing nothing, can compare attendee != object, didn't really see anything problematic

Emails are looking nice, nice job :)